### ISSUE TITLE
Search all schemas for a name

### ISSUE THREAD

**Issue by kleinux** [microsoft/vscode-mssql#893]
- MSSQL Extension Version: 1.0.0
- VSCode Version: 1.12.2
- OS Version: Windows 10 Enterprise

When writing a SQL statement the auto-complete options that appear only show suggestions from the default schema. Can we get an option to show matches from all schemas if no schema is supplied?

For example suppose I have tables:
Sale.OrderItem
Reference.OrderItemHistory

If I wrote "select * from Ord" I'd like to see both tables as suggestions while "select * from Sale.Ord" would only show the Sale.OrderItem table.

I hope this is the correct place to post this feature request and thanks for making such a great product.

**Comment by kevcunnane** [comment]
Thanks Chris, this is definitely the correct place to make feature requests. We will add this suggestion to our backlog and consider it in the future. We are mostly reusing the IntelliSense provider that comes from existing tools (SSMS & SSDT) so the fix would be in that code. It's very much valid however so we can see if this can be looked into sometime down the road.

**Comment by Kenobi-the-2nd** [comment]
I agree. This feature would be very helpful when working across multiple schemas. I consistently work across different schemas and databases and find myself writing queries often when testing my applications. It is tedious to have to prefix the table names with the schema each time. I hope that this feature can be added soon, as it would greatly increase productivity.

**Comment by SomeoneIsWorking** [comment]
Does this extension feature any searching capability at all? Can I quickly jump to a stored procedure by typing its name on a tiny database?

**Comment by croblesm** [comment]
Hi @kleinux,

Thanks again for the original suggestion! We appreciate you taking the time to share this idea with us.

This request has been open for quite some time without additional community feedback, and given recent changes to how IntelliSense and metadata handling work in the extension, we don’t currently plan to pursue this enhancement.

We’ll go ahead and close this issue to help keep the backlog focused on active requests. If there’s renewed interest or updated scenarios where cross-schema suggestions would be valuable, please feel free to open a new issue so we can revisit it with the current architecture in mind.
